src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -66,10 +64,8 @@ public MainViewModel(PowerToysRunSettings settings)
 
             _historyItemsStorage = new WoxJsonStorage<QueryHistory>();
             _userSelectedRecordStorage = new WoxJsonStorage<UserSelectedRecord>();
-            _topMostRecordStorage = new WoxJsonStorage<TopMostRecord>();
```

Comment:
Consider putting a comment in TopMostRecord.cs stating that it's not used anymore, or just removing that file altogether.

Reply:
removed file